tools/cli/admin.go

```diff
@@ -799,6 +803,55 @@ func newDBCommands() []cli.Command {
 				AdminDBScan(c)
 			},
 		},
+		{
+			Name:    "timers",
+			Aliases: []string{"timers"},
+			Usage:   "list timers",
+			Flags: append(getDBFlags(),
+				cli.IntFlag{
+					Name:  FlagLowerShardBound,
+					Usage: "lower bound of shard to scan (inclusive)",
+					Value: 0,
+				},
+				cli.IntFlag{
+					Name:  FlagUpperShardBound,
+					Usage: "upper bound of shard to scan (exclusive)",
+					Value: 16384,
+				},
+				cli.IntFlag{
+					Name:  FlagPageSize,
+					Usage: "page size used to query db executions table",
+					Value: 500,
+				},
+				cli.IntFlag{
+					Name:  FlagConcurrency,
+					Usage: "number of threads to handle scan",
+					Value: 1000,
+				},
+				cli.StringFlag{
+					Name:  FlagStartDate,
+					Usage: "start date",
+					Value: time.Now().UTC().Format(time.RFC3339),
+				},
+				cli.StringFlag{
+					Name:  FlagEndDate,
+					Usage: "end date",
+					Value: time.Now().UTC().Add(24 * time.Hour).Format(time.RFC3339),
```

Comment:
Just a suggestion, as it would require additional implementation effort.

In such occasions it usually much more convenient to deal with relative timestamps instead. For example `now + 1d` or `now + 2w`. There are libraries available for such support, e.g. https://github.com/karrick/tparse or you may choose to implement simplified version yourself.

Reply:
Thanks Vytautas, 

Will leave it as it is now, but your suggestion can be implemented in another PR. Let's figure it out, what will be usage pattern of this tool, first.